packages/grafana-e2e/src/flows/addPanel.ts

```diff
@@ -77,6 +85,11 @@ export const addPanel = (config?: Partial<AddPanelConfig>): any =>
       .get('button[title="Apply changes and go back to dashboard"]')
       .click();
 
+    e2e().wait('@chartData');
+
+    // Wait for RxJS
+    e2e().wait(500);
```

Comment:
seems brittle

Reply:
I don't understand. It doesn't kill it after 500ms, it simply waits. I don't know why this happens, but Cypress `screenshot()` causes RxJS to fail. What would logging a message accomplish?

Reply:
what happens if rxjs is not loaded?

Reply:
is there a way to determine what you are waiting for has loaded?

Reply:
Not that I know of. RxJS isn't a global.

Reply:
i'm ignorant here with cypress.  can you execute something that would test the existence of rxjs?

Reply:
This sounds very disturbing, I'd like to look into this closer but will not have time today. RxJs is our library for streaming data from the backendSrv to visualisations so we really don't know when it's completed.

Reply:
"Streaming" doesn't sound like a normal request, so perhaps it can be covered in the near future with https://github.com/cypress-io/cypress/issues/687

Reply:
> i'm ignorant here with cypress. can you execute something that would test the existence of rxjs?

Not that I can think of. We can `wait()` for requested URLs (currently only `XMLHttpRequest` and not `fetch()`).